api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -982,4 +977,12 @@ public DbUser updateRasLinkLoginGovStatus(String loginGovUserName) {
         dbUser,
         Agent.asUser(dbUser));
   }
+
+  /** Confirm that a user's profile is up to date, for annual renewal compliance purposes. */
+  @Override
+  public DbUser confirmProfile() {
+    final DbUser dbUser = userProvider.get();
+    dbUser.setProfileLastConfirmedTime(new Timestamp(clock.instant().toEpochMilli()));
+    return userDao.save(dbUser);
+  }
```

Comment:
This should translate into an `update users set profile_last_completion_time=? where user_id=?` without a transaction. I say without a transaction because there is no need for one and therefore no need to increase contention. Do you know if an easy way to verify this?

Reply:
This UserDao extends CrudRepository(Spring JPA) and will use default transaction configuration:  [Spring Docs](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#transactions)

Reply:
I have a very difficult time trusting anything that doesn't execute, such as documentation. I was curious if there was an easy way to view the SQL generated from this code to ensure it isn't creating an unnecessary transaction nor creating a race condition by having too many fields in the update clause, for example.

Reply:
You can run the test case with these properties uncommented and it will output all sql statements generated: https://github.com/all-of-us/workbench/blob/61bb9e20eb2421cafae8d825ae0a78210d28d909/api/src/test/resources/application.properties#L10-L15